README.md

```diff
@@ -9,14 +9,15 @@ Both are created, run and managed by ReDI School of Digital Integration. We're a
 
 ## Getting started for developers
 
-1. Make sure mongodb is installed your computer
-2. If you're re-seeding, drop the database files: `rm -rf mongodb-data/*`
-3. Run `yarn` in project root to install dependencies
-4. Run `yarn start:all` to boot all apps, _or_ a subset of apps using the `start:x` commands in package.json.
+1. Make sure [MongoDB](https://docs.mongodb.com/manual/installation/) is installed on your computer.
+2. If you're re-seeding, drop the database files: `rm -rf mongodb-data/*` and run `yarn seed` to seed the database.
+3. Make sure you're running the _v14.17.6_ version of Node locally when you're running the backend server. To do this, you can use [nvm](https://github.com/nvm-sh/nvm#node-version-manager), which allows you to select different versions of Node via the command line. Alternatively, we have added support for [Volta](https://docs.volta.sh/guide/understanding). So if you choose, you can use Volta, which sets project-defined tools automatically.
+4. Run `yarn` in project root to install dependencies.
+5. Run `yarn start:all` to boot all apps, _or_ a subset of apps using the `start:x` commands in package.json.
 
```

Comment:
I think people will freak out a bit with all the error messages thrown by the backend 😅 Can we include a note that people shouldn't worry about it?

Reply:
There is one on the line 18. Would you like another one or different wording?

Reply:
Oops, that's what happens when I only focus on the diff, not the whole README file. All good then :)